lib/v2/screens/profile_screens/guardians/select_guardian/interactor/select_guardians_bloc.dart

```diff
@@ -0,0 +1,26 @@
+import 'package:bloc/bloc.dart';
+import 'package:seeds/v2/datasource/remote/model/member_model.dart';
+import 'package:seeds/v2/screens/profile_screens/guardians/select_guardian/interactor/viewmodels/select_guardians_events.dart';
+import 'package:seeds/v2/screens/profile_screens/guardians/select_guardian/interactor/viewmodels/select_guardians_state.dart';
+
+/// --- BLOC:thi
+class SelectGuardiansBloc extends Bloc<SelectGuardiansEvent, SelectGuardiansState> {
+  SelectGuardiansBloc() : super(SelectGuardiansState.initial());
+
+  @override
+  Stream<SelectGuardiansState> mapEventToState(SelectGuardiansEvent event) async* {
+    if (event is OnUserSelected) {
+      var mutableSet = <MemberModel>{};
```

Comment:
Sets make it so we can select the same user more than one, 
But since our states are Immutability  "final" we need to copy them to mutable sets before updating them.